src/slices/AccountSlice.ts

```diff
@@ -206,16 +208,18 @@ export const loadAccountDetails = createAsyncThunk(
     let unstakeAllowanceV2 = BigNumber.from("0");
     let unstakeAllowance = BigNumber.from("0");
     let wrapAllowance = BigNumber.from("0");
-    let unwrapAllowance = BigNumber.from("0");
     let gOhmUnwrapAllowance = BigNumber.from("0");
     let poolAllowance = BigNumber.from("0");
-    let gOhmBalance = BigNumber.from("0");
```

Comment:
this is an interesting catch. I'm confused how `gohmBalance` got thrown in with a bunch of allowance checks, but...

can you go look at line 196 of the `migrateThunk` now? Was `gohmBalance` added here because we were incorrectly checking balances there? I think that line (196) should be changed to `getBalances`
[Line 196](https://github.com/OlympusDAO/olympus-frontend/blob/develop/src/slices/MigrateThunk.ts#L196)